modules/cnv-storage-wizard-fields-web.adoc

```diff
@@ -10,22 +10,20 @@
 |===
 |Name | Description
 
-|Create Disk
-|Create a new disk for the virtual machine.
+|Source
+|Select a blank disk for the virtual machine or choose from the options available: PXE, Container, URL or Disk. To select an existing disk, choose *Attach Disk* from a list of available PVCs or from a cloned disk and attach to the virtual machine.
```

Comment:
Because these are actual elements in the UI, bold them: `*PXE*, *Container*, *URL*, or *Disk*`

Reply:
The second sentence ("To select an existing disk,...") needs to be rewritten for clarity.

Reply:
Modified as follows:
"To select an existing disk and attach it to the virtual machine, choose *Attach Disk* from a list of available PersistentVolumeClaims (PVCs) or from a cloned disk."